make test fails on all branches

`make test` fails on all branches `feature/google-service`. Below is the command and the error message:

```
make test
go test -i $(go list ./... | grep -v 'vendor') || exit 1
tools/tools.go:5:2: import "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs" is a program, not an importable package
```

--- a/internal/provider/resource_bgp_session.go
+++ b/internal/provider/resource_bgp_session.go
@@ -243,7 +243,7 @@ func resourceBgpSessionDelete(ctx context.Context, d *schema.ResourceData, m int
 	}
 	session, err := c.GetBgpSessionBy(cID.(string), connCID.(string), bgpSettingsUUID.(string))
 	if err != nil {
-		return diag.Errorf("could not find BGP session associated with the provided Cloud Router ID", err)
+		return diag.Errorf("could not find BGP session associated with the provided Cloud Router ID: %v", err)
 	}
 	sessionToDisable := session.BuildNewBgpSessionInstance()
 	sessionPrefixes, err := c.ReadBgpSessionPrefixes(bgpSettingsUUID.(string))

--- a/tools/tools.go
+++ b/tools/tools.go
@@ -1,6 +1,4 @@
 package tools
 
-import (
-	// document generation
-	_ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"
-)
+// document generation
+// _ "github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs"

